npc/other/Global_Functions.txt

```diff
@@ -434,7 +434,14 @@ function	script	F_ShuffleNumbers	{
 //== Function F_MesColor ===================================
 // Function to colorize npc dialog without having to memorize the color code
 // Examples:
-//    mes callfunc("F_MesColor", C_BLUE) +"This message is now in BLUE";
+//    mes F_MesColor(C_BLUE) +"This message is now in BLUE.";
```

Comment:
this work with mesf? should include parentheses like mes() 
`mesf("%sThis message is now in BLUE.", F_MesColor(C_BLUE));`

Reply:
the problem is this line is inside the `*mes` area, not in the `*mesf` area